src/Surfnet/ServiceProviderDashboard/Application/Assembler/ServiceStatusAssembler.php

```diff
@@ -78,30 +80,22 @@ class ServiceStatusAssembler
 
     /**
      * The mapping of the legend
-     *
-     * @var array
      */
-    private $legend = [
+    private array $legend = [
         self::SERVICE_STATUS_SUCCESS => '#67a979',
         self::SERVICE_STATUS_IN_PROGRESS => '#f6aa61',
         self::SERVICE_STATUS_INFO => '#d1d2d6',
     ];
 
-    /**
-     * @var ServiceStatusDto
-     */
-    private $serviceStatusDto;
+    private readonly ServiceStatusDto $serviceStatusDto;
 
     /**
      * ServiceStatusAssembler constructor.
```

Comment:
Feel free to also remove this irrelevant constructor text

Reply:
done